src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,18 +3,64 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_LENGTH_MAX = 10;
+    private K[] keysStorage;
+    private V[] valuesStorage;
+    private int size;
+
+    public StorageImpl() {
+        this.keysStorage = (K[]) new Object[ARRAY_LENGTH_MAX];
+        this.valuesStorage = (V[]) new Object[ARRAY_LENGTH_MAX];
+    }
+
+    public K[] getKeysStorage() {
+        return keysStorage;
+    }
+
+    public void setKeysStorage(K[] keysStorage) {
+        this.keysStorage = keysStorage;
+    }
+
+    public V[] getValuesStorage() {
+        return valuesStorage;
+    }
+
+    public void setValuesStorage(V[] valuesStorage) {
+        this.valuesStorage = valuesStorage;
+    }
 
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < ARRAY_LENGTH_MAX; i++) {
+            if (key == keysStorage[i] && valuesStorage[i] != null
+                    || key != null
+                    && key.equals(keysStorage[i])) {
```

Comment:
```suggestion
            if (key == keysStorage[i]  || key != null && key.equals(keysStorage[i])) {
```
if we iterate arr from 0 to < size, we will be able to simplify this check